webapp/templates/base.html

```diff
@@ -42,6 +42,7 @@
             <li><a href="/">Home</a></li>
             <li><a href="{% url 'view_products' %}">Producten</a></li>
             <li><a href="{% url 'docs.document_overview' %}">Documenten</a></li>
+            <li><a href="{% url 'member_list' %}">Leden</a></li>
```

Comment:
Can you change it to `Ledenlijst` here too?

Reply:
Oeps, yes!